src/auth.ts

```
@@ -1,61 +1,67 @@
 import { JSON_HEADER } from "@lib/constants/shared.constant";
-import { LoginResponse } from "@lib/types/auth";
 import { NextAuthOptions } from "next-auth";
 import Credentials from "next-auth/providers/credentials";
+import jwt from "jsonwebtoken";
```

ESLint

[error] 'jwt' is defined but never used.

(@typescript-eslint/no-unused-vars)